client/utils/assessment.js

```diff
@@ -109,6 +110,10 @@ export const schemas = {
       response: yup.string().trim().min(1).max(200).required()
     })
   }),
+  TS: yup.object().shape({
+    text: yup.string().trim().min(1).required(),
+    correct: yup.array().of(yup.mixed().required()).min(1)
```

Comment:
Isn't this array of ranges/pairs? [`yup#mixed`](https://github.com/jquense/yup#mixed) is effectively _any_ type?